pkg/kubelet/config/config.go

```diff
@@ -519,6 +519,7 @@ func copyPods(sourcePods []*v1.Pod) []*v1.Pod {
 		pod, err := api.Scheme.Copy(source)
 		if err != nil {
 			glog.Errorf("unable to copy pod: %v", err)
+			continue
```

Comment:
@yujuhong should we skip the pod or start returning an error here? What should we do this for other functions in this file (e.g. `MergedState`) that have the same pattern?